rest-api-spec/src/main/resources/rest-api-spec/api/ml.put_trained_model_definition_part.json

```diff
@@ -0,0 +1,38 @@
+{
+  "ml.put_trained_model_definition_part":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/put-trained-model-definition-part.html",
+      "description":"Adds part of a trained model definition"
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_ml/trained_models/{model_id}/definition/{part_num}",
```

Comment:
The `{part_num}` in the path must match the key in `parts: {...}` so should be `{part}`